roborazzi-compose-preview-scanner-support/src/main/java/com/github/takahirom/roborazzi/RoborazziPreviewScannerSupport.kt

```diff
@@ -19,13 +19,24 @@ fun ComposablePreview<AndroidPreviewInfo>.captureRoboImage(
 
 @ExperimentalRoborazziApi
 interface ComposePreviewTester<T : Any> {
+  class Options(
```

Comment:
This would be the biggest change for users in this PR. This change is a breaking change for users using ComposePreviewTester. This is for reducing future breaking changes when we add options, as adding parameters to the interface would break users' implementations. Do you have any opinions about this?

Reply:
I think it does not require a breaking change.

You could simply extend the interface with
```kotlin
fun getPreviewOptions(): Options
```
and then use that new method in `ComposePreviewTester<AndroidPreviewInfo>.previews()`

Or I am missing something?